nestkernel/connection.h

```diff
@@ -150,6 +150,12 @@ class Connection
   void set_status( const DictionaryDatum& d, ConnectorModel& cm );
 
   /**
+   * Check syn_spec dictionary for parameters that are not allowed for this
+   * connection. Will issue warning or throw error if a parameter is found.
+   */
```

Comment:
See above, the base class doc should be here. Mention that the base class implementation does nothing and that classes requiring checks need to override with their own method.